src/main/java/farmsystem/backend/domain/trade/service/RankingService.java

```diff
@@ -0,0 +1,32 @@
+package farmsystem.backend.domain.trade.service;
+
+import farmsystem.backend.domain.profile.entity.Profile;
+import farmsystem.backend.domain.profile.entity.ProfileType;
+import farmsystem.backend.domain.profile.repository.ProfileRepository;
+import farmsystem.backend.domain.trade.dto.response.AssetRankingResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class RankingService {
+
+    private final ProfileRepository profileRepository;
+
+    // 자산 랭킹 조회
+    public List<AssetRankingResponse> getAssetRanking() {
+        List<Profile> profiles = profileRepository.findByType(ProfileType.LIVE);
+
+        profiles.sort(Comparator.comparingLong(Profile::getBalance).reversed()); // 정렬
+
+        return profiles.stream()
+                .map(profile -> AssetRankingResponse.of(profile.getMember().getUsername(), profile.getBalance()))
```

Comment:
이런식으로 하면 서비스단 좀 간단하게 가져갈 수 있을 것 같습니당
당장은 안바꿔도 상관 없을 것 같아요!!

```suggestion
                .map(profile -> AssetRankingResponse.from(profile)
```